tests/playwright/sanity/modules/playing-cards/widget.test.js

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Playing Cards @playing-cards', () => {
+	test( 'Playing Cards widget', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.openNewPage();
+
+		await test.step( 'Add widget', async () => {
+			await editor.addWidget( 'playing-cards' );
+
+			const widget = await editor.getPreviewFrame().locator( EditorSelectors.playingCards.container );
+
+			await expect( await widget.count() ).toBe( 1 );
```

Comment:
Alternatively you could use `expect.soft()` to get an error without stopping your test at this point. The test will still fail at the end of the test. This technique is useful when regenerating screenshots on CI without waiting 30min for each.

In this case I'd suggest just using `.first()` as noted above because the `expect` here is testing for side effects in Playwright functions.` addWidget()` has been very stable for some time.